ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/FixtureExtensions.kt

```diff
@@ -0,0 +1,27 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.uitests.fixtures
+
+import com.intellij.remoterobot.fixtures.CommonContainerFixture
+import com.intellij.remoterobot.fixtures.ComponentFixture
+import com.intellij.remoterobot.fixtures.JTextFieldFixture
+import com.intellij.remoterobot.search.locators.byXpath
+import com.intellij.remoterobot.stepsProcessing.step
+import java.time.Duration
+
+fun ComponentFixture.rightClick() = step("Right click") {
+    runJs("robot.rightClick(component);")
+}
+
+fun CommonContainerFixture.pressOk() = findAndClick("//div[@text='OK']")
+fun CommonContainerFixture.pressDelete() = findAndClick("//div[@text='Delete']")
+
+fun CommonContainerFixture.findAndClick(xPath: String) = findByXpath(xPath).click()
+fun CommonContainerFixture.findByXpath(xPath: String) = find<ComponentFixture>(byXpath(xPath), Duration.ofSeconds(5))
+
+fun CommonContainerFixture.fillSingleTextField(text: String) = step("Fill single text field with $text") {
+    find<JTextFieldFixture>(byXpath("//div[@class='JTextField']"), Duration.ofSeconds(5)).text = text
+    // Wait for whatever changed to populate (enable OK button etc), otherwise we might continue too quickly
```

Comment:
Example?

Reply:
Fill in S3 bucket name, it clicks before the OK button is enabled then fails the test

Reply:
Hmmm..

There are probably better / safer ways but we can revisit it later